<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upgradePrompts</class>
 <widget class="QDialog" name="upgradePrompts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>版本升级</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:1px;
border-color:rgb(240,240,240);
border-radius:4px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_windowTitle" native="true">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>110</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/png/updateBackground.png);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>158</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font-weight:bold;
font-size:30px;
color: rgba(77, 132, 255, 1);
</string>
           </property>
           <property name="text">
            <string>版本升级</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>158</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_close">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color:rgb(242, 113, 113); 
image: url(:/png/closeW.png)15 0 0 0;
}
QPushButton { 
image: url(:/png/close.png)15 0 0 0;
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QScrollBar:vertical{ 
    width:8px;  
    border-style:flat;
    border-radius: 4px;
    border:0px;
     background: rgb(255, 255, 255);
} 
QScrollBar::handle:vertical{ 
    background: rgb(200, 200, 200);
    border-radius: 4px;
    width:8px; 
    min-height:50px; 
    border-style:flat;
}
QScrollBar::handle:vertical::hover{ 
    background: rgb(130, 130, 130);
    border-radius: 4px;
    width:8px; 
}
QScrollBar::handle:vertical::pressed{ 
    background: rgb(130, 130, 130);
    border-radius:4px;
    width:8px; 
}
QScrollBar::sub-page:vertical {
     background: rgb(255, 255, 255);
border-style:flat;
}
QScrollBar::add-page:vertical {
     background: rgb(255, 255, 255);
border-style:flat;
}
QScrollBar::add-line:vertical{
     background: rgb(255, 255, 255);
}
QScrollBar::sub-line:vertical {
     background: rgb(255, 255, 255);
}
QScrollBar:horizontal{ 
    height:8px;  
    border-style:flat;
    border-radius: 4px;
    border:0px;
    background: rgb(255, 255, 255);
} 
QScrollBar::handle:horizontal{ 
    background: rgb(200, 200, 200);
    border-radius: 4px;
    height:8px; 
    min-width:50px; 
    border-style:flat;
}
QScrollBar::handle:horizontal::hover{ 
    background: rgb(130, 130, 130);
    border-radius: 4px;
    height:8px; 
}
QScrollBar::handle:horizontal::pressed{ 
    background: rgb(130, 130, 130);
    border-radius:4px;
    height:8px; 
}
QScrollBar::sub-page:horizontal {
     background: rgb(255, 255, 255);
    border-style:flat;
}
QScrollBar::add-page:horizontal {
     background: rgb(255, 255, 255);
    border-style:flat;
}
QScrollBar::sub-line:horizontal {
     background: rgb(255, 255, 255);
}
QScrollBar::add-line:horizontal{
     background: rgb(255, 255, 255);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label_info">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:16px;
color: rgba(54, 65, 89, 1);</string>
           </property>
           <property name="text">
            <string>MCU仿真工具已发布！</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:14px;
color: rgba(54, 65, 89, 1);
</string>
           </property>
           <property name="text">
            <string>本次更新内容有：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="styleSheet">
            <string notr="true">font-size:13px;
color: rgba(54, 65, 89, 1);
border-style:nono;
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_force">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight:bold;
font-size:14px;
color: rgba(230, 0, 0, 1);
</string>
           </property>
           <property name="text">
            <string>当前更新为强制更新，如取消，程序将无法使用。</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>25</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(140,140,140);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(140,140,140);
border-radius:4px;
padding-left:15px;
padding-right:15px;</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ok">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font-size:14px;
border-style:solid;
border-width:2px;
background-color: rgb(47, 105, 235);
border-color:rgb(47, 105, 235);
border-radius:4px;</string>
          </property>
          <property name="text">
           <string>立即下载</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
